.github/workflows/build.yml

```diff
@@ -35,16 +32,29 @@ jobs:
           ~/.ghc-wasm/add_to_github_path.sh
           popd
 
-      - name: checkout
+      - name: Checkout Pandoc-wasm
         uses: actions/checkout@v4
 
-      - name: checkout
+      - name: Checkout Pandoc
         uses: actions/checkout@v4
         with:
-          repository: haskell-wasm/pandoc
-          ref: wasm
+          repository: jgm/pandoc
+          ref: main
```

Comment:
If you are going to put a version number on these builds, shouldn't they also be pinned to a specific commit (probably a release tag) for the underlying Pandoc version too?

Reply:
@alerque Yes, you are right. I am trying to figure out how to best do this. It should probably be easy to get it to build with the most recent pandoc version and we probably need another digit to the number to show the version of the pandoc-wasm package. So `3.6.3.x` instead of just `3.6.3`. If you have a proposal of how to do the versioning in the most standard complying and simple way - I'm all for it.

Reply:
Just adding a segment will make it hard to parse because Pandoc follows [PVP](https://pvp.haskell.org/) and has a variable number of segments. I think you're going to need a different segment operator that plays nice with distro versioning (I think `+` is the most robust option). The question is probably does it make more sense to version *this* project first, then append the relevant Pandoc version, or the other way around? e.g. `pandoc-wasm-3.6.3+0.1` or `pandoc-wasm-0.1+3.6.3`. I think the latter probably makes more sense but it depends on the expected release channels and use workflows I guess. I don't really have a handle on that.

Reply:
@alerque OK, as someone working mainly with JS/TS in browsers, I would expect it to become available in npm. But I'm open to others needing it other places - maybe? I like both of your versioning proposals, and unless there is reason not to do so, I'd go for the second one then.

Reply:
@alerque This seems to now be working with tagged versions, etc. . Only thing remaining is put it it on npm. As I assume that @TerrorJack or @tweaf or @alerque will want to have control over the npm repository after merging this PR (or writing something similar), I will not add that part.

Reply:
@alerque It looks like npm does not allow this versioning scheme. It only allows 3 digit semver. So I split the two numbering apart. It's still really easy to update the pandoc version though.